documentation/Admin/validator_recovery.md

```diff
@@ -62,48 +62,55 @@ In order to restore the Axelar proxy key used by the Vald process, you can use t
 
 ### Recovering Tofnd state
 
-In order to restore the node's key shares, you can use the `--tofnd-mnemonic` and `--recovery-info` flags with `join/launchValidator.sh` as follows:
+If you want to reset your tofnd (e.g. on a new machine, after unexpected data loss, etc), you will have to recover your tofnd state. Tofnd's state consists of the following:
+1. **your private key**: Internal tofnd key used to encrypt your recovery data. This private key is derived from a mnemonic that is generated automatically when tofnd is executed for the first time on your machine. You should have stored this mneminic safely, since it is the only passphrase that can be used to recover your key shares.
+2. **your key shares**: Data that is generated when you participate into a keygen and is used to perform sign. 
 
-```
-./join/joinTestnet.sh --tofnd-mnemonic /path/to/tofnd/mnemonic/ --recovery-info /path/to/recovery/file/
-```
-
-If you also need to 
-
-### Recover with tofnd binary
+Each time you participated in a keygen, your key shares were encrypted and stored on the blockchain. This means that you can easily fetch your shares, but you must have your private key (i.e. launch tofnd with your mnemonic) to successfully decrypt them.
 
-In order to recover, you will need to execute tofnd in *import* mode. To do that, use the `import` command-line option:
+#### Running tofnd in a containerized environment
 
-* `import`: Adds a new mnemonic from file *.tofnd/import*; Succeeds when there is no other mnemonic already imported, fails otherwise.
+In order to restore tofnd's private key and your key shares, you can use `join/launchValidator.sh` with the `--tofnd-mnemonic` and `--recovery-info` flags with as follows: 
 
-All tofnd mnemonic options can be displayed by running
 ```
-cargo run -- -h
+./join/joinTestnet.sh --tofnd-mnemonic <mnemonic file> ---recovery-info <recover json file>
 ```
 
-#### Example
-An exercise for restoring your tofnd key using your mnemonic is the following:
-1. Create an new empty *.tofnd* folder, and write your mnemonic into a file under the name *import*. Put this file in *./tofnd/import*. 
+1. `<mnemonic file>`: A file that contains your mnemonic passphrase  
+2. `<recover json file>`: The recovery information in json format you receive by executing 
     ```
-    mkdir .tofnd && cd .tofnd
-    touch import
-    # write your mnemonic at the `import` file
+    axelard q tss recover $(axelard keys show validator --bech val -a) btc-master btc-secondary --output json > recovery.json
     ```
-2. Start tofnd using the *import* option
+    after attaching to your validator container (see section [Recover Data](#Recovery_Data)).
+
+#### Running tofnd as binary
+
+If you are running a tofnd binary, follow the steps below:
```

Comment:
I would capitalize each point in the enumeration

Reply:
done